### PR TITLE
Add simplecov for rspec coverage reporting

### LINKED ISSUE #94: Coverage HTML Reporter for Rails Too?
Jest is configured to generate coverage reports (HTML). Should we have rubocop do the same?

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Closes #94 

coverage-simplecov

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

**Comment by eckdanny-osi** [comment]
this is also an enabler for #60

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+require 'simplecov'
+SimpleCov.start do
```
Comment (edgibbs) [inline spec/spec_helper.rb]: You'll need one for rails_helper as well
```ruby
SimpleCov.start 'rails' do
  coverage_dir 'coverage/ruby'
end

Comment (edgibbs) [inline spec/spec_helper.rb]: And I think there's a story for this somewhere in the backlog.

Comment (eckdanny-osi) [reply spec/spec_helper.rb]: added in 5b3ee6ce9108db05f3ec79e1c4663dbc8251b18d

Comment (eckdanny-osi) [reply spec/spec_helper.rb]: [CASE-1088](https://osi-cwds.atlassian.net/browse/CASE-1088)

**Comment by eckdanny-osi** [comment]
@edgibbs ready for re-review

**Pull request merged**